crates/wysiwyg/src/dom/dom_struct.rs

```diff
@@ -445,7 +445,7 @@ where
             let sub_handle = handle.sub_handle_up_to(i);
             if sub_handle.is_root() {
                 continue;
-            } else if let Some(node) =
+            } else if let Some::<&DomNode<S>>(node) =
```

Comment:
Not sure why but this was't coming back with the generic applied, it was being inferred as <DomNode<Unknown>>, which was causing an error due to a type mismatch when trying to assign to current.